### PR TITLE
add all main files which create popup

### PR DESCRIPTION
**Pull request by BogdanGapon**
### css & html popup

[Demo](https://bogdangapon.github.io/HTML-and-CSS-practice-kottans/) |
[Code base](https://github.com/BogdanGapon/frontend-2022-homeworks/tree/html-css-popup/submissions/BogdanGapon/HTML%20%26%20CSS%20popup)

### CODE REVIEW

**Comment by HannaSyn** [comment]
Good work! 
Please, see my comments below

**Comment by HannaSyn** [comment]
According the task, all the  interactive elements  should have `:focus` state, please, make it for Menu icon and More button too.

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            <li class="header-nav__item popup-item">
+              <label class="custom-checkbox" for="check"></label>
+              <input
+                class="checkbox visually-hidden"
+                type="checkbox"
+                id="check"
+              />
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <div class="popup-flex-wrapper">
```
Comment (HannaSyn) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:30]: Please, make all icon list semantic - use `ul>li` tags

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+                <input
+                  class="checkbox-more visually-hidden"
+                  type="checkbox"
+                  name="checkbox_2"
+                  id="labe_id"
+                />
+                <label for="labe_id" class="label-more is-hidden"> More </label>
+                <div class="next-popup-wrapper">
+                  <div class="popup-flex-wrapper">
```
Comment (HannaSyn) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:137]: This icon list should be semantic too - use `ul>li` tags

File: submissions/BogdanGapon/HTML & CSS popup/style.css [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  text-decoration: none;
+  color: rgb(17, 15, 15);
+  -webkit-transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+  transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+}
```
Comment (HannaSyn) [inline submissions/BogdanGapon/HTML & CSS popup/style.css:89]: Please, format css file - it needs to be white-space after each block

File: submissions/BogdanGapon/HTML & CSS popup/style.css [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  z-index: 10;
+}
+.checkbox-more:checked ~ .next-popup-wrapper {
+  display: block;
+}
+
+.checkbox-more:checked + .is-hidden {
+  visibility: hidden;
+  pointer-events: none;
+}/*# sourceMappingURL=style.css.map */
```
Comment (HannaSyn) [inline submissions/BogdanGapon/HTML & CSS popup/style.css:235]: Please, avoid unnecessary comments and add white-space in the end of the file - [reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

**Review by HannaSyn — CHANGES_REQUESTED** [review]

File: submissions/BogdanGapon/HTML & CSS popup/index.html (hunk 1 above)
Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/index.html:30]: ahah it`s a joke?? previous mentor say to delete all ul>>li

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+                      src="./img/calendar.png"
+                      alt="Calendar"
+                      width="45"
+                      height="45"
+                    />Calendar</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/chat.png"
+                      alt="Chat"
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:58]: Please, delete all alt values in img tag in nav or popUP. If image is used for decoration and don't share any information to user, its alt attribute should be ampty without any symbols or spacing.

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <div class="popup-flex-wrapper">
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/apps.png"
+                      alt="Colorful circle"
+                      width="45"
+                      height="45"
+                    />Colorful circle</a
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:37]: It is really great that you have fount that attributes width and height could be passed to img tag. But they are using, when css styles are don't. So if you use css styles this attributes using is incorrect for html and css separation logic.
Please, move this styles into css.
image

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+                  >
+                </div>
+
+                <input
+                  class="checkbox-more visually-hidden"
+                  type="checkbox"
+                  name="checkbox_2"
+                  id="labe_id"
+                />
+                <label for="labe_id" class="label-more is-hidden"> More </label>
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:135]: Please, delete white spaces. `<label for="labe_id" class="label-more is-hidden">More</label>`

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+position: absolute;
+top:65px;
+right:50px;
+width: 350px;
+height: 400px;
+padding: 30px 15px 30px;
+border:1px solid rgba(128, 128, 128,0.4);
+box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+background-color: #fff;
+overflow:auto;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+        &.link-img:hover {
+        box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+        }
+}
+
+.visually-hidden {
+position: absolute;
+appearance: none;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+justify-content: flex-end;
+align-items: center;
+gap: 25px;
+}
+
+.header-nav__item  {
+display: flex;
+justify-content: center;
+align-items: center;
+flex-basis: 30px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 11]
```diff
@@ -0,0 +1,161 @@
+.header {
+padding:20px;
+background-color: #fff;
+}
+
+.header-nav__list {
+display: flex;
+justify-content: flex-end;
+align-items: center;
+gap: 25px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 12]
```diff
@@ -0,0 +1,161 @@
+.header {
+padding:20px;
+background-color: #fff;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/base/_container.scss [hunk 13]
```diff
@@ -0,0 +1,4 @@
+.container {
+max-width: 2860px;
+margin: 0 auto;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/base/_container.scss:3]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+flex-basis: 30px;
+}
+
+.header-nav__link {
+display: flex;
+justify-content: center;
+align-items: center;
+text-decoration: none;
+color: rgb(17, 15, 15);
+transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+
+    &:hover,
+    &:focus { 
+    color: rgb(170, 63, 63);
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:30]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 16]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+        &.link-img:hover {
+        box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 17]
```diff
@@ -0,0 +1,161 @@
...
+    content: " ";
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    width: 30px;
+    height: 30px;
+    background-image: url(../img/popup-button.png);
+    background-repeat: no-repeat;
+    background-size: 30px 30px;
+    cursor: pointer;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:56]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 18]
```diff
@@ -0,0 +1,161 @@
...
+width: 350px;
+height: 400px;
+padding: 30px 15px 30px;
+border:1px solid rgba(128, 128, 128,0.4);
+box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+background-color: #fff;
+overflow:auto;
+
+    .checkbox:checked ~ & {
+    display: block;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:78]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 19]
```diff
@@ -0,0 +1,161 @@
...
+    }
+}
+
+.popup-list__link {
+width: calc((100% - 120px) / 3);
+text-decoration: none;
+color: #000;
+text-align: center;
+
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:124]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 20]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.next-popup-wrapper{
+display: none;
+position: absolute;
+top: 500px;
+right: 0px;
+padding: 30px 15px 50px;
+background-color: #fff;
+z-index: 10;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 21]
```diff
@@ -0,0 +1,161 @@
...
+
+    .checkbox-more:checked ~ & {
+    display: block;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 22]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+}
+
+.checkbox-more:checked + .is-hidden {
+visibility: hidden;
+pointer-events: none;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Please, fix spacing

File: submissions/BogdanGapon/HTML & CSS popup/style.css [hunk 23]
```diff
@@ -0,0 +1,235 @@
...
+.header-nav__link {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/style.css]: You should use 1 tab spacing for all rules

File: submissions/BogdanGapon/HTML & CSS popup/style.css [hunk 24]
```diff
@@ -0,0 +1,235 @@
...
+.next-popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 500px;
+  right: 0px;
+  padding: 30px 15px 50px;
+  background-color: #fff;
+  z-index: 10;
+}
+.checkbox-more:checked ~ .next-popup-wrapper {
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/style.css]: Maybe` [selector]+[selector]` is more siutable for this case. Please check it.

File: submissions/BogdanGapon/HTML & CSS popup/style.css [hunk 25]
```diff
@@ -0,0 +1,235 @@
...
+  position: absolute;
+  top: 65px;
+  right: 50px;
+  width: 350px;
+  height: 400px;
+  padding: 30px 15px 30px;
+  border: 1px solid rgba(128, 128, 128, 0.4);
+  -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+          box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+  background-color: #fff;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/style.css]: You should use css variables if value is used more then twice.
Please, fix such cases in all files.

**Comment by Roophee** [comment]
Good job!!!
Please, fix comments issues.

File: submissions/BogdanGapon/HTML & CSS popup/index.html (hunk 6 above)
Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/index.html]: "But they are using, when css styles are don't." I don`t use css styles for img (width and height) only responsible styles like  max-width: 100%; and that is all.

Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/index.html]: But you use css files. This what i mean.  You can't swith between this two cases as you want. If you are using css files, you hade better not use this attributes.

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss (hunk 9 above)
Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: i did`t know i need to fix all spacing when css file do it automatically when i write any property and use enter i see 1 tab from property to value and its really good seeing this better appearance: none; than this appearance:none;

Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: So if you don't. Please fix your code formatter in your vsCode. Code style is very important. 
You can add Prettier extention in code redactor.

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 26]
```diff
@@ -0,0 +1,132 @@
...
+                  <ul class="popup-item">
+                    <li class="popup-list__list">
+                      <a href="">
+                        <img src="./img/promo.png" alt=""/>Video
+                        </a>
+                    </li>
+                    <li class="popup-list__list">
+                      <a href="">
+                        <img src="./img/search.png" alt=""/>Search
+                        </a>
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:106]: Sorry, bro. Lets  close the issue about width and height for img tag. Have serched about this  and this is my bad. Please revert  all  changes related to this comment. You really made searching good job.
https://jakearchibald.com/2022/img-aspect-ratio/

Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/index.html]: it`s okay, thank you ) i revert all changes related to this comment.

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 27]
```diff
@@ -0,0 +1,176 @@
...
+  cursor:pointer;
+
+  .checkbox-more:focus ~ & {
+    border:1px solid #000;
+  }
+
+}
+
+.next-popup-wrapper {
+  display:none;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:160]: Why all your css rules don't have space after `:`?
` display: none;`
Maybe, formatter issue?
Can you fix it?

Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:160]: Just say that you seriously ??  You write Please, fix spacing i DELETE all spacing after css :,i mean i sit and delete all spaces which you say and now you say not delete? I don`t understund what i need to do, you write more 10 comments with "Please, fix spacing", i fixed all spacing and now i need to remote it again??

Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:160]: Screenshot_2

Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: I mean tab spacing in previous comments. You had something like that:
`.main {
display: flex;
  align-items: center;
  -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
          box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
outline: 2px solid red
 } `
 
 I asked you to make lite like this:
 `.main {
   display: flex;
   align-items: center;
   box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
   -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
   outline: 2px solid red 
   }`
   
   But not delete space after `:` symbol.
   Thanks you for your patient, sorry for misunderstanding.

Comment (BogdanGapon) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: I remove it again

Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss]: Can you ping me in Telegram?

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 28]
```diff
@@ -0,0 +1,162 @@
...
+  content: " ";
+  position: absolute;
+  top: 55px;
+  z-index: 10;
+
+  display: none;
+  width: 20px;
+  height: 20px;
+  border-top: 1px solid rgba(128, 128, 128, 0.4);
+  border-left: 1px solid rgba(128, 128, 128, 0.4);
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:98]: Why you dont use your variables?

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 29]
```diff
@@ -0,0 +1,162 @@
...
+.custom-checkbox {
+  position: relative;
+
+  &::before {
+    content: " ";
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    width: 30px;
+    height: 30px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:48]: We can create variable --size-30: 30px;

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 30]
```diff
@@ -0,0 +1,162 @@
...
+    background-image: url(../img/popup-button.png);
+    background-repeat: no-repeat;
+    background-size: 30px 30px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:51]: use var(--size-30)

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 31]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+.popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 65px;
+  right: 50px;
+  width: 350px;
+  height: 400px;
+  padding: 30px 15px 30px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:67]: use var(--size-30)

File: submissions/BogdanGapon/HTML & CSS popup/sass/base/_base.scss [hunk 32]
```diff
@@ -0,0 +1,34 @@
...
+  display: block;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+}
+
+.section {
+  padding: 50px 0;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/base/_base.scss:33]: we can create var --size-50: 50px;

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 33]
```diff
@@ -0,0 +1,162 @@
...
+
+.visually-hidden:focus ~ .custom-checkbox::before {
+  outline: 2px solid var(--text-color);
+}
+
+.popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 65px;
+  right: 50px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:64]: use var(--size-50)

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 34]
```diff
@@ -0,0 +1,162 @@
...
+    border: 1px solid var(--text-color);
+  }
+}
+
+.next-popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 500px;
+  right: 0px;
+  padding: 30px 15px 50px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:150]: use var(--size-30) 15px var(--size-50)

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 35]
```diff
@@ -0,0 +1,162 @@
...
+.header-nav__list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 25px;
+}
+
+.header-nav__item {
+  @include flex;
+  flex-basis: 30px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:15]: use var(--size-30);

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 36]
```diff
@@ -0,0 +1,162 @@
+.header {
+  padding: 20px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:2]: we can create var  --size-20: 20px;

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 37]
```diff
@@ -0,0 +1,162 @@
...
+
+.span {
+  content: " ";
+  position: absolute;
+  top: 55px;
+  z-index: 10;
+
+  display: none;
+  width: 20px;
+  height: 20px;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:96]: use var(--size-20)

File: submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss [hunk 38]
```diff
@@ -0,0 +1,162 @@
+.header {
+  padding: 20px;
+  background-color: #fff;
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss:3]: we can create --black: #fff; or --default-bg-color or --bg-color

File: submissions/BogdanGapon/HTML & CSS popup/index.html (hunk 1 above)
Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/index.html]: Please, delete all alt attrtibutes **values** in popUP and nav icons (img tags). If image is used for decorations and don't have any sanse or information for user, alt attribute should be empty, without any symbols or spacing.

Comment (Roophee) [reply submissions/BogdanGapon/HTML & CSS popup/index.html]: Values of alt attributes.

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 39]
```diff
@@ -0,0 +1,121 @@
...
+            <div class="popup-wrapper">
+              <ul class="popup-item">
+                <li class="popup-list__list">
+                  <a href="">
+                    <img src="./img/apps.png" alt="" width="45" height="45" />Colorful circle
+                  </a>
+                </li>
+                <li class="popup-list__list">
+                  <a href="">
+                    <img src="./img/blogs.png" alt="Blog" width="45" height="45" />Blog
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:36]: This alt is not empty

File: submissions/BogdanGapon/HTML & CSS popup/index.html [hunk 40]
```diff
@@ -0,0 +1,121 @@
...
+                      <img src="./img/videos.png" alt="" width="45" height="45" />Video
+                    </a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="header-nav__item">
+            <a href="" class="header-nav__link link-img">
+              <img src="./img/bell.png" alt="Bell" width="30" height="30" /></a>
```
Comment (Roophee) [inline submissions/BogdanGapon/HTML & CSS popup/index.html:113]: This alt is not empty too

**Review by Roophee — APPROVED** [review]

**Review by HannaSyn — APPROVED** [review]

**Pull request merged**